d2-test-api/src/main/java/com/linkedin/d2/balancer/clients/TestClient.java

```diff
@@ -0,0 +1,150 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.linkedin.d2.balancer.clients;
+
+import com.linkedin.common.callback.Callback;
+import com.linkedin.common.util.None;
+import com.linkedin.data.ByteString;
+import com.linkedin.r2.filter.R2Constants;
+import com.linkedin.r2.message.RequestContext;
+import com.linkedin.r2.message.rest.RestRequest;
+import com.linkedin.r2.message.rest.RestResponse;
+import com.linkedin.r2.message.rest.RestResponseBuilder;
+import com.linkedin.r2.message.stream.StreamRequest;
+import com.linkedin.r2.message.stream.StreamResponse;
+import com.linkedin.r2.message.stream.StreamResponseBuilder;
+import com.linkedin.r2.message.stream.entitystream.ByteStringWriter;
+import com.linkedin.r2.message.stream.entitystream.EntityStreams;
+import com.linkedin.r2.transport.common.bridge.client.TransportClient;
+import com.linkedin.r2.transport.common.bridge.common.TransportCallback;
+import com.linkedin.r2.transport.common.bridge.common.TransportResponseImpl;
+import java.util.Map;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+
+public class TestClient implements TransportClient
+{
+  public static final int DEFAULT_REQUEST_TIMEOUT = 500;
+  public StreamRequest streamRequest;
+  public RestRequest restRequest;
+  public RequestContext restRequestContext;
+  public Map<String, String> restWireAttrs;
+  public TransportCallback<StreamResponse> streamCallback;
+  public TransportCallback<RestResponse>   restCallback;
+  public ScheduledExecutorService _scheduler;
+
+  public boolean                         shutdownCalled;
```

Comment:
I know this file is not written by you, but this variables naming is not consistent with other class variables, can you fix this by this chance?

Reply:
Sure, may as well while I'm at it

Reply:
Actually I decided to keep it as-is because these variables are public.